cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -341,10 +350,25 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
           statsOnlyResultTable()
         }
       }
+      section("Altering roles", "administration-security-roles-alter", "enterprise-edition") {
```

Comment:
should it be named `Modifying roles` to be consistent with the user one (`Modifying users`)?